src/MOI_wrapper.jl

```diff
@@ -134,6 +134,18 @@ function _allocate_constraint(cone::ConeData, f, s::MOI.ExponentialCone)
     cone.ep += 1
     ci
 end
+constroffset(cone::ConeData, ci::CI{<:MOI.AbstractFunction, <:MOI.PowerCone}) = cone.f + cone.l + cone.q + cone.s + cone.ep + ci.value
+function _allocate_constraint(cone::ConeData, f, s::MOI.PowerCone)
+    ci = length(cone.p)
+    push!(cone.p, s.exponent)
+    ci
+end
+constroffset(cone::ConeData, ci::CI{<:MOI.AbstractFunction, <:MOI.DualPowerCone}) = cone.f + cone.l + cone.q + cone.s + cone.ep + ci.value
+function _allocate_constraint(cone::ConeData, f, s::MOI.DualPowerCone)
+    ci = length(cone.p)
+    push!(cone.p, s.exponent)
```

Comment:
It seems to me that with the current implementation, the `DualPowerCone` will be treated as a `PowerCone`

Reply:
You're right, here is the corrected code.